java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -43,9 +53,12 @@ class DevToolsService : Service() {
                 cont.resume(Unit) {}
             }
         }
+        scope.launch { initialize() }
     }
 
     override fun onBind(intent: Intent): IBinder? {
+        binder.send(storageService?.getStorageKeys() ?: "")
+        devToolsServer.addOnOpenWebsocketCallback { storageService?.getStorageKeys() ?: "" }
```

Comment:
To me this line looks like it's setting up a callback when a websocket is opened, and just reading the storage keys but not doing anything with them... is that right? Is it supposed to also be sending the keys to the client?

Reply:
Updated (Also see comment below).